src/routes.ts

```diff
@@ -43,6 +43,10 @@ const routes = [
     path: '/login',
     component: () => import('./routes/login'),
   },
+  {
+    path: '/newrealm',
```

Comment:
Nit: existing URLs with multiple "words" use dashes (e.g. `saved-reports`)

```suggestion
    path: '/new-realm',
```